contracts/token/ERC20/CappedToken.sol

```diff
@@ -11,7 +11,7 @@ contract CappedToken is MintableToken {
 
   uint256 public cap;
 
-  constructor(uint256 _cap) public {
+  constructor(uint256 _cap, address[] _minters) public MintableToken(_minters) {
```

Comment:
Just to be explicit: the convention will be to have all role bearer arrays at the end of the argument list.

Reply:
This should probably be in the ownership mini readme.